### PR TITLE
Fix stats and usage with multipe testsuites

### PR DESCRIPTION
**Pull request by Pharb**

### CODE REVIEW

File: test/mock-results.js [hunk 1]
```diff
@@ -4,6 +4,15 @@ module.exports = function(stats){
   var data = {
     testsuites: [
       {
+        _attr: {
+          name: "Mocha Tests",
+          tests: "2",
+          failures: "1",
+          timestamp: stats.start.toISOString().substr(0, stats.start.toISOString().indexOf('.')),
+          time: "0.002"
```
Comment (clayreimann) [inline test/mock-results.js]: Should this really be hard coded? Won't that make the test brittle?

**Comment by clayreimann** [comment]
Another rebase, and maybe add another suite to the test to demonstrate that we don't duplicate output anymore?

**Comment by Pharb** [comment]
The output was not duplicated, the report was wrong when using multiple suites.
It works better now, and the reported data is now correct.

File: test/mock-results.js (hunk 1 above)
Comment (Pharb) [inline test/mock-results.js]: Suite time is now the sum of all testcase durations belonging to the suite.

**Comment by Pharb** [comment]
I'm fixing the conflict.
